index.html

```diff
@@ -168,6 +168,24 @@ <h3 class="language-title">Skills</h3>
                 </div>
             </div>
         </section>
+        <section class="section-contuct-me">
+            <div class="container">
+                <h2 class="contuct-me__heading">Contact me</h2>
+                <p class="contuct-me__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+                <input type="text" class="contuct-me__name-input" placeholder="Enter Your Name">
+                <input type="email" class="contuct-me__email-input" placeholder="Enter Your Email">
+                <textarea name="Enter Your massages Here" id="" cols="30" rows="10" class="contuct-me__textarea" placeholder="Enter your massages here"></textarea>
+                <a href="#" class="contuct-me__btn">Get in touch</a>
```

Comment:
I think it would be better to use a ```form``` tag for this part

Reply:
Nice job being suggestive and polite👍 But we can even do better by add a reason why the change is requested😉 in other words, let's know why you want the block of code to be wrapped in a `form` tag😉 don't forget we have learnt in previous lessons on[ "how to give a code review"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) that we should Remember to explain why you request a change - it might be obvious for you but not for someone else😉. Kindly edit the comments and add an understandable reason for requesting the change😉👍